service/src/test/java/org/ehrbase/dao/access/jooq/FolderAccessTest.java

```diff
@@ -363,7 +363,22 @@ public void shouldUpdateExistingFolder() throws Exception {
         fa2.setFolderName("modifiedName");
         fa2.setFolderNArchetypeNodeId("modifiedArchetypeNodeId");
         fa2.setIsFolderActive(false);
-        fa2.setFolderDetails(DSL.field(DSL.val("{\"s\": \"modifiedValue\"}") + "::jsonb"));
+        ItemStructure is = new ItemStructure() {
```

Comment:
`ItemStructure` is an abstract class, right? From the openEHR RM perspective it is, see: https://specifications.openehr.org/releases/RM/latest/data_structures.html#_item_structure_class

So when you always assign a list style structure here, you could also directly create an non-abstract `ItemList` archie class instance. This would allow a leaner construction, too: https://github.com/openEHR/archie/blob/master/openehr-rm/src/main/java/com/nedap/archie/rm/datastructures/ItemList.java#L36